moped-editor/src/views/projects/projectView/ProjectComponents/ComponentMapToolbar.js

```diff
@@ -45,7 +45,12 @@ export default function ComponentMapToolbar({
           />
         </Box>
         <Box mr={2}>
-          <MapAlert message={message} severity={severity} isOpen={isOpen} />
+          <MapAlert
+            message={message}
+            severity={severity}
+            isOpen={isOpen}
+            onClose={onClose ? onClose : null}
```

Comment:
Passing this prop gives us the X icon to close the error message. I'm open to different ways of showing the "can't find the linked component" error message. Would appreciate feedback on how that works since it requires users to dismiss the error.